lib/generate.js

```diff
@@ -193,16 +193,11 @@
         output += parser.customTokens.map(function (token) { return "declare var " + token + ": any;\n" }).join("")
         output += parser.body.join('\n');
         output += "\n";
-        output += "interface NearleyToken {";
-        output += "  value: any;\n";
-        output += "  [key: string]: any;\n";
-        output += "};\n";
-        output += "\n";
```

Comment:
Wait… are you sure this should be deleted? Isn't `NearleyToken` referenced in the type signature for `next()` a couple lines below?

Reply:
um yeah not sure what i was thinking sorry. somehow i ran the wrong test